### DISCUSSION TITLE
Welcome to gmt-china.org Discussions!

### DISCUSSION THREAD

**Discussion by seisman** [gmt-china/gmt-china.org#283] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by core-man** [comment]
Hello~

**Comment by liuzhumei** [comment]
Mark~

**Comment by kanglcn** [comment]
Hello! Will the "Discussions" replace the QQ groups?

**Reply by liuzhumei** [reply]
Please don't
cause most of our Chinese users cannot log in github.com

**Reply by kanglcn** [reply]
Got you. Thanks.
